community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/outputs.tf

```diff
@@ -55,6 +55,7 @@ output "nodeset_tpu" {
 }
 
 output "checksum" {
-  description = "Checksum of all files written to the bucket."
-  value       = local.checksum
+  description = "Checksum of config written to the bucket."
+  # wrap into `md5` to get nice representation suitable for use in labels
```

Comment:
`nit`: I would suggest being more explicit here about converstion from base 64 to hex. I believe issue is b64 contains characters that are not allowed in label values.

https://cloud.google.com/compute/docs/labeling-resources#requirements